src/main.js

```diff
@@ -1,10 +1,11 @@
-const { app, BrowserWindow, ipcMain, autoUpdater } = require("electron");
+const electron = require("electron");
+const { app, BrowserWindow, protocol } = electron;
+const { autoUpdater } = require("update-electron-app");
```

Comment:
The autoUpdater will (i hope) automatically publish the new releases when they are published on github